eox_nelp/views.py

```diff
@@ -34,3 +40,26 @@ def info_view(request):
         json.dumps(response_data),
         content_type="application/json"
     )
+
+
+class NelpMFEConfigView(MFEConfigView):
+    """
+    Provides an API endpoint to get the MFE_CONFIG from site configuration with
+    nelp extra fields.
+    """
+
+    @method_decorator(cache_page(settings.MFE_CONFIG_API_CACHE_TIMEOUT))
+    def get(self, request):
+        """
+        Return the MFE configuration by NELP(adding custom nelp fields).
+        """
+        base_get_response = super().get(request)
+        mfe_config_dict = json.loads(base_get_response.content)
+        theme_options = theming.options('THEME_OPTIONS')
+        interactive_color = theming.options('interactive_color')
+        theme_additions = {
+            'THEME_OPTIONS': theme_options,
+            'CUSTOM_PRIMARY_COLOR': {'primary': interactive_color},
```

Comment:
it almost official, that this should be  `png-color-primary-base`

Reply:
[cd92192](https://github.com/eduNEXT/eox-nelp/pull/17/commits/cd92192612f88edeed0a34be248fd28c4dec7ca6)